mathjax3-ts/ui/menu/Menu.ts

```diff
@@ -436,33 +461,55 @@ export class Menu {
                         this.command('Reset', 'Reset to defaults', () => this.resetDefaults())
                     ]),
                     this.submenu('Accessibility', 'Accessibility', [
-                        this.submenu('Explorer', 'Explorer', [
-                            this.checkbox('Active', 'Active', 'explorer'),
-                            this.rule(),
-                            this.submenu('Highlight', 'Highlight', this.radioGroup('highlight', [
-                                ['None'], ['Hover'], ['Flame']
-                            ]), true),
-                            this.submenu('Background', 'Background', this.radioGroup('background', [
-                                ['Blue'], ['Red'], ['Green'], ['Yellow'], ['Cyan'], ['Magenta'], ['White'], ['Black']
-                            ])),
-                            this.submenu('Foreground', 'Foreground', this.radioGroup('foreground', [
-                                ['Black'], ['White'], ['Magenta'], ['Cyan'], ['Yellow'], ['Green'], ['Red'], ['Blue']
-                            ])),
-                            this.rule(),
+                        this.checkbox('Activate', 'Activate', 'explorer'),
+                        this.submenu('Speech', 'Speech', [
                             this.checkbox('Speech', 'Speech Output', 'speech'),
-                            this.checkbox('Subtitles', 'Subtitles', 'subtitles'),
+                            this.checkbox('Subtitles', 'Subtities', 'subtitles'),
+                            this.checkbox('Braille', 'Braille', 'braille'),
+                            this.checkbox('View Braille', 'View Braille', 'viewbraille'),
```

Comment:
I'm wondering if `Braille subtitles` (and perhaps `Speech subtitles` above) would be more parallel, and so easier to understand?  I don't have strong feelings about it either way.

Reply:
Homogenised this.